src/main/scala/sigmastate/Values.scala

```diff
@@ -588,6 +588,10 @@ object Values {
     def propBytes: Value[SByteArray] = SigmaPropBytes(SigmaPropConstant(sb))
   }
 
+  implicit class BoolValueOps(b: BoolValue) {
```

Comment:
@greenhat It is better to make all implicit classes `extends AnyVal` to avoid allocations

Reply:
Done.